packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/extract-apple-private-key.js

```diff
@@ -0,0 +1,17 @@
+const PRIVATE_KEY_REGEX = /(-+BEGINPRIVATEKEY-+)(.+[^-])(-+ENDPRIVATEKEY-+)/;
```

Comment:
can we make this a .ts file and use `export function` rathern that module.exports?

Reply:
I tried making this a .ts file at first, but kept getting build errors. I'll have to go back and try to remember the reason again.